go/teams/chain.go

```diff
@@ -854,8 +866,7 @@ func (t *TeamSigChainPlayer) addInnerLink(
 			hasMembers(true),
 			hasParent(false),
 			hasInvites(false),
-			hasSubteam(false),
-			hasInvites(false))
+			hasSubteam(false))
```

Comment:
If settings are not supported for this link then add `hasSettings(false)`. Same for all the other `case`s.